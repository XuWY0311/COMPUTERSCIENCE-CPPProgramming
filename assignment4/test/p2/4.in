40
112 111 112 111 111 112 112 112 112 112 112 111 112 111 111 111 111 111 112 111 112 111 111 112 111 111 111 112 111 111 112 112 112 111 112 112 112 111 111 112