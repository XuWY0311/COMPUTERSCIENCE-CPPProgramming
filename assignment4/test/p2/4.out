{ 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 111, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112, 112 }
